### PR TITLE
Implement SE-0193 - part 1

### PR DESCRIPTION
**Pull request by slavapestov**
This PR renames the attributes to their final names, but not does not yet add the change where `@inlinable` implies `@usableFromInline` when applied to an `internal` declaration. I'd like to make that change in the next PR and test the renaming on its own.

### CODE REVIEW

**Comment by slavapestov** [comment]
@airspeedswift @milseman @lorentey @lancep Just a heads-up that I'm planning on landing this shortly, and it touches a lot of stdlib files. Do you have any big changes in flight I should wait for?

**Comment by slavapestov** [comment]
@swift-ci Please test

**Comment by slavapestov** [comment]
@swift-ci Please test source compatibility

**Comment by slavapestov** [comment]
@swift-ci Please test source compatibility

**Review by jrose-apple — COMMENTED** [review]
No big changes coming on my end, but one of the commits in here is not a change we want!

File: lib/Sema/ResilienceDiagnostics.cpp [hunk 1]
```diff
@@ -116,23 +116,8 @@ bool TypeChecker::diagnoseInlinableDeclRef(SourceLoc loc,
...
-  }
-
-  if (isDefaultArgument) {
-    diagnose(D, diag::resilience_decl_declared_here,
-             D->getDescriptiveKind(), D->getFullName());
-  } else {
-    diagnose(D, diag::resilience_decl_declared_here_versioned,
-             D->getDescriptiveKind(), D->getFullName());
-  }
+  diagnose(D, diag::resilience_decl_declared_here,
```
[the remark this answers is not in the document]
Comment (slavapestov) [reply lib/Sema/ResilienceDiagnostics.cpp]: This change only affects the wording in the diagnostic - you could already reference usableFromInline things. We don’t serialize Exprs so there’s nothing to print, and a common complaint when we rolled out the default argument change was people wanted to reference non-public things. What do you think?

**Comment by slavapestov** [comment]
I’ll fix the other comments and drop the default argument change for now. We can discuss it separately.

File: include/swift/AST/Decl.h [hunk 2]
```diff
@@ -2244,12 +2245,12 @@ class ValueDecl : public Decl {
   ///
   /// \sa getFormalAccessScope
   AccessLevel getFormalAccess(const DeclContext *useDC = nullptr,
-                              bool respectVersionedAttr = false) const {
+                              bool respectUsableFromInline = false) const {
```
[the remark this answers is not in the document]
Comment (slavapestov) [reply include/swift/AST/Decl.h]: isUsageFromInline? It’s kind of weird isn’t it

**Pull request merged**